functional_test/pet_store_client.py

```diff
@@ -132,4 +132,21 @@ def signup_user(self, user):
 
         return self.session.request('POST', url, self.headers, json.dumps(user))
 
+    def update_user(self, user):
+        """
+        Updates a user, returning the updated user response
+        :param user:
+        :return:
+        """
+        url = urljoin(self.index_url, '/users')
+
+        return self.session.request('PUT', url, self.headers, json.dumps(user))
+
+    def list_users(self, page_size=10, offset=0):
+        """
+        Returns a list of users
+        :return:
+        """
+        url = urljoin(self.index_url, "/users?pageSize={0}&offset={1}".format(page_size, offset))
 
+        return self.session.request('GET', url, self.headers)
```

Comment:
here we need a new line at the end of the file, otherwise we get these red little arrows